examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -360,6 +361,25 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             break;
 
           case 'pointmap':
+
+            var pointsJson = (JSON.stringify({
+              dataset: parameters.dataset,
+              filter: getFilter(parameters, defaultNonSamplingDayRange, parameters.geoIds),
```

Comment:
`defaultNonSamplingDayRange` is too big for the db. Please use a smaller day filter value, e.g., 2 or 4 days, before we fully support the limit in the middleware layer.
plus, these number should be configurable.